app/components/widgets/CopyableAddress.scss

```diff
@@ -1,14 +1,30 @@
 .component {
+  * {
+    vertical-align: middle;
```

Comment:
This causes a regression on the address at the top of the "Your address page"

### Before

<img width="597" alt="image" src="https://user-images.githubusercontent.com/2608559/61990036-93dd6b00-b073-11e9-99a3-ae2f379ab652.png">

### After

<img width="621" alt="image" src="https://user-images.githubusercontent.com/2608559/61990040-9fc92d00-b073-11e9-8ec7-604a26ad1615.png">

Reply:
I've updated the vertical alignment:

<img width="628" alt="vertical_alignment_hover" src="https://user-images.githubusercontent.com/1937074/62088164-47a14f00-b25c-11e9-9587-98a7bbd15432.png">

<img width="632" alt="vertical_alignment_address" src="https://user-images.githubusercontent.com/1937074/62088103-13c62980-b25c-11e9-93a7-27e33b399d43.png">